slo-libsonnet/error-burn.libsonnet

```
@@ -3,9 +3,10 @@ local errors = import 'errors.libsonnet';
 {
   errorburn(param):: {
     local slo = {
-      metric: error 'must set metric for errorburn',
-      selectors: error 'must set selectors for errorburn',
-      errorBudget: error 'must set errorBudget for errorburn',
+      alertName: error 'must set alert name for error burn',
```

We could have a fallback here. Essetially
```suggestion
      alertName: 'ErrorBudgetBurn'
```

This makes it from the outside the same as right now, but if people wish they can change it.